docs/tutorials/getting-started.md

```diff
@@ -161,6 +161,7 @@ The `<py-config>` tag can be used as follows:
   lang = "python"
 </py-config>
 ```
+Note: `[[runtimes]]` is a TOML table. Make sure this is the last item within a py-config, as the properties created after it goes into the runtimes object.
```

Comment:
Little thing - probably wants to be “_go_ into the runtimes object”?

Reply:
goes to go!